test/tests/functional/pbs_alps_inventory_check_hook.py

```diff
@@ -131,7 +136,8 @@ def test_cray_login_nodes(self):
         """
         Test log when no nodes with vntype 'cray_login' are present.
         """
-        now = time.time()
+        mc = ""

```

Comment:
Because codacy was looking this as an issue if the variable is accessed before initialize. 
Updated.